arlo-client/src/components/CreateAudit.tsx

```diff
@@ -41,6 +54,35 @@ const CreateAudit = ({ history }: RouteComponentProps<ICreateAuditParams>) => {
       toast.error(err.message)
     }
   }
+
+  const [elections, setElections] = useState<IElections>({ elections: [] })
+
+  const getStatus = useCallback(async (): Promise<IElections> => {
+    try {
+      const list: IElections | IErrorResponse = await api(`/elections`)
+      if (checkAndToast(list)) {
+        return { elections: [] }
+      } else {
+        return list
+      }
+    } catch (err) {
+      toast.error(err.message)
+    }
+    return { elections: [] }
+  }, [])
+
+  const updateElections = useCallback(async () => {
+    const list = await getStatus()
+    setLoading(true)
+    setElections(list)
+    setElectionId(list.elections.length ? list.elections[0].id : '')
+    setLoading(false)
+  }, [getStatus])
+
+  useEffect(() => {
+    updateElections()
```

Comment:
This setup is prone to race conditions. See this section on [Traditional Approaches vs. Suspense](http://reactjs.org/docs/concurrent-mode-suspense.html#traditional-approaches-vs-suspense) for an illustration of why. In _this particular case_ it probably won't be an issue, but it's an approach prone to error and should be avoided.